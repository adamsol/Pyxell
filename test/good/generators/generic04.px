
func* reversed(a) def
    yield ...a by -1

print [...reversed([1, 2])]
print [...reversed("test")]
