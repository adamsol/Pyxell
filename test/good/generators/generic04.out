[2, 1]
['t', 's', 'e', 't']
